bundles/org.openhab.binding.salus/README.md

```diff
@@ -75,11 +75,12 @@ removed.
 
 ### `salus-it600-device` Channels
 
-| Channel                     | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
-|-----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| it600-temp-channel          | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
+| Channel                    | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
+|----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| it600-temp-channel         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
```

Comment:
According to the it600.xml this channel id is different. Looks like you added the type id, while it should bne the channel id here:
```suggestion
| temperature         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
```